operators/stackgres/1.5.0/manifests/stackgres.clusterserviceversion.yaml

```diff
@@ -0,0 +1,913 @@
+                  test -n "$OPERATOR_NAME"
+                  OPERATOR_UID="$(kubectl get operators.operators.coreos.com "$OPERATOR_NAME" --template '{{ .metadata.uid }}')"
+                  test -n "$OPERATOR_UID"
+
+                  cat << EOF | kubectl apply -f -
+                  apiVersion: stackgres.io/v1
+                  kind: SGConfig
+                  metadata:
+                    name: stackgres-operator
+                  spec:
+                  $SGCONFIG_SPEC
+                    clusterOwnerReferences:
+                    - apiVersion: operators.coreos.com/v1
+                      kind: Operator
+                      name: $OPERATOR_NAME
+                      uid: $OPERATOR_UID
+                  EOF
+                image: quay.io/ongres/kubectl@sha256:20ba4329217b11c82ff8665c1f81599a75f8d4319e9546b6fd6ec86a12608914
+                imagePullPolicy: IfNotPresent
+                name: default
+                resources:
+                  limits:
+                    cpu: "1"
+                    memory: 512Mi
+                  requests:
+                    cpu: 10m
+                    memory: 8Mi
+                securityContext:
+                  allowPrivilegeEscalation: false
+                  capabilities:
+                    drop:
+                    - ALL
+                  runAsGroup: 1000
+                  runAsNonRoot: true
+                  runAsUser: 1000
+              securityContext:
+                runAsNonRoot: true
+              serviceAccountName: stackgres-helm-controller-manager
+              terminationGracePeriodSeconds: 10
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - patch
+        serviceAccountName: stackgres-helm-controller-manager
+    strategy: deployment
+  installModes:
+  - supported: false
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - postgresql
+  - postgres
+  - database
+  - sql
+  - rdbms
+  - open source
+  - ongres
+  links:
+  - name: StackGres Web
+    url: https://stackgres.io
+  - name: StackGres Docs
+    url: https://stackgres.io/doc
+  maintainers:
+  - email: stackgres@ongres.com
+    name: OnGres
+  maturity: stable
+  minKubeVersion: 1.18.0
+  provider:
+    name: OnGres
+    url: https://ongres.com
+  version: 1.5.0
```

Comment:
I am not sure if I should add `CSV.spec.replaces` field with value `stackgres.v1.4.3` or the pipeline will add it to the catalog using semver ordering.